src/lib/HST/Util/Selectors.hs

```diff
@@ -5,17 +5,22 @@ module HST.Util.Selectors
     expFromUnguardedRhs
     -- * Pattern Names
   , getAltConName
+  , getIdentifiers
```

Comment:
Consider to rename this to `findIdentifiers` to communicate the linear complexity of the function.